ykrt/src/compile/jitc_yk/codegen/x64/mod.rs

```diff
@@ -137,6 +137,7 @@ struct Assemble<'a> {
     ra: LSRegAlloc<'a>,
     tloop_gp_reg_states: Option<[lsregalloc::RegState; 16]>,
     tloop_fp_reg_states: Option<[lsregalloc::RegState; 16]>,
+    loop_start_locs: Vec<VarLocation>,
```

Comment:
Let's add a doc comment to fields by default unless they're blindingly obvious.

Reply:
[7170a3a](https://github.com/ykjit/yk/pull/1381/commits/7170a3a8cdbbdc740e24647e4f5194bc0ff29c04)

Reply:
I assume some methods that used these fields are now unused and are dead code?